prow/plugins/trigger/generic-comment.go

```diff
@@ -29,6 +31,24 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
+var (
+	testHelpRe          = regexp.MustCompile(`(?m)^(/test\s*\?|/list-tests)$`)
```

Comment:
@cblecker WDYT about the `/test ?`/`/list-tests` commands?

Reply:
I'm not a fan of `/list-tests` as we don't use the list keyword anywhere. I don't mind `/test` with no keyword, or `/test ?` as they follow the current format.

Reply:
Thank you for the review. Fixed.